src/SFA.DAS.Tasks.API.Types/Enums/ApprenticeshipEmployerType.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace SFA.DAS.Tasks.API.Types.Enums
+{
+    [Flags]
+    public enum ApprenticeshipEmployerType
+    {
+        None = 0,
+        Levy = 1,
+        NonLevy = 1 << 1,
+        All = ~None
```

Comment:
Not sure you need the All flag?

Reply:
We will need the all flag as the default value for the client as suggested above. Also All (-1) is the default value set on the column in sql.

Reply:
Can you not apply the Default as (ApprenticeshipEmployerType.Levy | ApprenticeshipEmployerType.NonLevy) this would give a value of All so no need for a specific enum value

Reply:
Can do, however it becomes tricky to decide on default value on the column. Currently it is set to `-1` which will apply to both Levy and Non-levy as well as any types introduced in the future. If we decide to use `(ApprenticeshipEmployerType.Levy | ApprenticeshipEmployerType.NonLevy)` then the default value will have to be `3` that will encompass `Both` but then any future changes will not be accommodated.

Reply:
I see, ok leave as is for now in that case